tests/test_mv_interface.py

```diff
@@ -6,7 +6,6 @@
 import os
 import signal
 
-import pylab
```

Comment:
I don't think it's such a big issue for the tests. Importing `pcdsdevices` and having it select a backend is the most annoying part.

Reply:
You're probably correct, but I wanted to be thorough.